### PR TITLE
Add basic layout navbar and router

### PR DESCRIPTION
**Pull request by Ewoskyx**
### First Layout

In this update I did;

- Add React Router and Navbar links
- Add folder structure and Redux setup
- Add components according to the original design
- Add styles according to the original design

### CODE REVIEW

File: src/Pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import Books from '../components/Books/Books';
+import Categories from './Categories';
+
+const Home = () => (
+  <>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Books />} />
+      <Route path="/categories" element={<Categories />} />
+    </Routes>
```
Comment (julie-ify) [inline src/Pages/Home.js]: - [x] Good job so far 🔥.  However, let's move everything concerning the route to the `index.js` file as it is not a good practice to have the routes in one of the components except `App` anyway. But it is a requirement to keep the route logic in the `index.js` file 😄.

File: src/components/Books/Books.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  Chapter, UpdateButton,
+} from './BooksElements';
+
+const Books = () => (
+  <>
+    <BookWrapper>
+      <BookDivLeft>
+        <Category>Action</Category>
+        <Title>The Hunger Games</Title>
+        <Author>Suzanne Collins</Author>
```
Comment (julie-ify) [inline src/components/Books/Books.js]: - [x] Let's create a new component called `BookList` that renders a list of books using the 'Books' component.
- [x] Let's pass the book data as a prop to the `Books` component and render the title and the author

File: src/components/Books/Books.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import NewBook from '../NewBook/NewBook';
+import {
+  BookWrapper, BookDivLeft, BookDivRight,
+  BookDivCenter, ButtonsContainer, Gauage, PercentageSpan, StatusSpan,
+  Category, Title, Author, BooksButton, CurrentChapter,
+  Chapter, UpdateButton,
+} from './BooksElements';
```
Comment (julie-ify) [inline src/components/Books/Books.js]: - [x] Nice implementation you have here 🔥. Keep rocking.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Ewoskyx,

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books/Books.js (hunk 2 above)
Comment (Ewoskyx) [reply src/components/Books/Books.js]: I totally agree with this :) Tbh, I planned to do that in the next step but doing it at the beginning looks like a best practice. Thank you :)

File: src/components/Books/Books.js (hunk 3 above)
Comment (Ewoskyx) [reply src/components/Books/Books.js]: Thank you for your kind comment 😃  ❤️ 🤝

File: src/Pages/Home.js (hunk 1 above)
Comment (Ewoskyx) [reply src/Pages/Home.js]: Yes, I had doubts about that. Now it is clear, I will do this change. Thank you 😃 👍

**Review by dasileker — APPROVED** [review]
## **Hi @Ewoskyx,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**